docs/integration-tests.md

```diff
@@ -0,0 +1,67 @@
+# Integration Tests
+
+* GCP Project
+* Create a service account with relevant access - See [Refer to IAM Credentials](Home.md#iam-credentials)
+* Most of the tests require a VM with Java pre-installed at `/usr/bin/java`.
+  There are one or two which don't require java pre-installed, as they supply `startup-script` to gcloud apis, and install `java` on a plain linux Debian image.
+  * You can create an image with `java` preinstalled as,
```

Comment:
Nicer to have the Maven project do this for you, say with Packer (checking whether the VM image already exists in the project and creating it on demand), either in a `@BeforeClass`, or more tediously in the POM by running some `exec:exec` goals before and after the IT phase.

Reply:
Currently linked back to here in the integration test notes
integration test PR https://github.com/jenkinsci/google-compute-engine-plugin/pull/495